assets/js/util.js

```diff
@@ -293,13 +305,62 @@ function callApy(options, endpoint) {
     var requestUrl = window.location.protocol + window.location.hostname +
         window.location.pathname + '?' + $.param(requestOptions.data);
 
+    if(!installationNotificationShown) {
```

Comment:
Whoops, I mistook the context in which this gets run. Can we move this check into `displayUnobtrusiveWarning` (which we should rename to `displayInstallationNotification`) - you can make it an early return. Then it only needs to happen in one place.